config/database.yml

```diff
@@ -0,0 +1,85 @@
+# PostgreSQL. Versions 9.3 and up are supported.
```

Comment:
ignore this file. Because we have sample config

Reply:
@SKH-WTAG bhai, done.